### ISSUE TITLE
Unable to preview pattern markdown / duplicate buttons

### ISSUE THREAD

**Issue by ryanchristo** [ushahidi/platform-pattern-library#135]
### Expected behaviour
- There should be only one "show html" button for each pattern.
- When I click the "show html" button, I can preview the markdown.

### Actual behaviour
- There are two "show html" buttons that overlap for each pattern.
- When I click either of the "show html" buttons, nothing happens.

<img width="1095" alt="screen shot 2017-06-22 at 5 51 54 pm" src="https://user-images.githubusercontent.com/12519942/27430812-381e106c-5774-11e7-80ad-89f45ffe7b50.png">

**Comment by ryanchristo** [comment]
I noticed that in the scripts file, in addition to loading the handlebars script, the hbLoadLayout() function is being invoked. When I removed this, everything seems to be working and the issue is fixed but I wanted to check and see if there was another reason for the function being invoked here before submitting a pull request.

**Comment by ryanchristo** [comment]
Updated pull request ushahidi/platform-pattern-library#136 with changes focuses on the insertion of the pattern-sample elements rather than when and how many times load layout is called.

**Comment by rjmackay** [comment]
@ryanchristo I'm still seeing this on http://preview.ushahidi.com/platform-pattern-library/develop/
Any ideas?

**Comment by ryanchristo** [comment]
@rjmackay I cleared my local files, pulled down the latest `develop` branch, ran `npm install` then `gulp` and it works as expected. Then I did the same for the latest `master` branch but I had to run `gulp build` before running `gulp` in order to get it to work. Is the preview running off the `master` branch? What's the build process when deployed?

`develop` branch / `master` branch running on local webserver:

<img width="467" alt="screen shot 2017-07-28 at 9 22 55 am" src="https://user-images.githubusercontent.com/12519942/28699946-d057ccba-7376-11e7-8e6a-764acfedf608.png">

preview running remotely

<img width="477" alt="screen shot 2017-07-28 at 9 23 03 am" src="https://user-images.githubusercontent.com/12519942/28699953-da89910a-7376-11e7-8882-8d5e9f48bc77.png">
